draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -193,6 +196,33 @@ MUST expose a different mDNS name for each address.
 
 ### Implementation Guidance
 
+#### Registration and Responses
+
+If the ICE agent skips both probing and announcement sending, the ICE agent
```

Comment:
The ICE agent is expected to answer queries regardless of announcements, right? Perhaps this paragraph should just be removed.

Reply:
Yes, I feel this is scoped to generic mDNS rules and we can leave it out.